src/entries/popup/pages/home/Activity/ActivitiesList.tsx

```diff
@@ -44,6 +44,22 @@ import { ActivityTypeIcon } from './ActivityTypeIcon';
 import { ActivityValue } from './ActivityValue';
 import { NoActivity } from './NoActivity';
 
+function getNftChangesTag(transaction: RainbowTransaction): string | undefined {
+  const { asset, changes } = transaction;
+  const nftChangesAmount =
+    changes?.reduce((count, c) => {
+      if (c && asset?.address === c.asset.address && c.asset.type === 'nft') {
+        const numericValue = Number.isInteger(Number(c.value))
```

Comment:
Do you mean to use `quantity` here? Also unexpectedly not a wei value for this type

Reply:
nice catch! but the issue lies deeper, I will ticket this out, see https://github.com/rainbow-me/browser-extension/blob/220aaa6826a80b7f7564470e58afccc156acc060/src/core/utils/transactions.ts#L303